### PR TITLE
feat: add db option and remove a deprecated option 

### PR DESCRIPTION
**Pull request by suguru03**
Related to: https://github.com/keroxp/deno-redis/issues/30

The key prefix option is a little bit complicated, I will send a small one first.

### CODE REVIEW

File: redis_test.ts [hunk 1]
```diff
@@ -128,6 +128,13 @@ test(async function testConcurrent() {
   assertEquals(c, "c");
 });
 
+test(async function testDbOption() {
+  await redis.set("exists", "aaa");
+  const client = await connect({ ...addr, db: 1 });
```
Comment (keroxp) [inline redis_test.ts]: Add test pattern when db=0

File: redis.ts [hunk 2]
```diff
@@ -1557,44 +1558,27 @@ export type RedisConnectOptions = {
...
-        hostname,
-        port
-      });
-    }
+export async function connect({
+  hostname,
+  port,
+  tls,
+  db,
+}: RedisConnectOptions): Promise<Redis> {
```
Comment (keroxp) [inline redis.ts]: Please don't delete string args from signature. This is out of scope change on this PR. We should delete it in the single PR someday.

File: redis_test.ts (hunk 1 above)
Comment (suguru03) [reply redis_test.ts]: Sure

File: redis.ts (hunk 2 above)
Comment (suguru03) [reply redis.ts]: Make sense.
If you are planning to delete it by the next minor release, I would like to do it before adding new options. Can I make a PR for it?

Comment (keroxp) [reply redis.ts]: Aah, I've understood what you thought about. There are no way to accept db options with string params, right? If so, now we can add those change into this PR.

**Comment by keroxp** [comment]
@suguru03 PR can be merged on adding tests!

**Comment by suguru03** [comment]
Sure, I will add the test case. 🙂

**Comment by suguru03** [comment]
Added! 😄
